src/runner.py

```diff
@@ -169,8 +195,10 @@ class cmd_run(Command):
     ]
 
     def __call__(self, configs, args):
-        Runner(configs, args).run()
-        return True
+        if not args.node_id and not args.git_commit:
+            print("Either --node-id or --git-commit is required")
```

Comment:
Shouldn't this be printed on stderr?

Reply:
Good point, I've fixed that now.  Also did this in #60 when printing errors from exceptions.